src/views/virtualmachines/actions/components/VirtualMachineMigration/tabs/VirtualMachineMigrationDestinationTab.tsx

```diff
@@ -44,10 +50,11 @@ const VirtualMachineMigrationDestinationTab: FC<VirtualMachineMigrationDestinati
                 {defaultStorageClassName === storageClass ? t('(default)') : ''}
               </>
             ),
+            isDisabled: !allowedStorageProfilesNames?.includes(storageClass),
             value: storageClass,
           }))}
           popperProps={{
-            appendTo: () => document.getElementById('virtual-machine-migration-modal'),
+            appendTo: () => document.getElementById(POPPER_CONTAINER_ID),
```

Comment:
1. why we are using a constant ID here instead of the default (`document.body`) ?
2. who creates/owns that element?